### DISCUSSION TITLE
Allow the quick summary to start as expanded

### DISCUSSION THREAD

**Discussion by 12th-devs** [PranshulGG/WeatherMaster#264] [Ideas]
Basically the title but I would love to see this as I open the app to read this quite often

**Comment by PranshulGG** [comment]
> Basically the title but I would love to see this as I open the app to read this quite often

@Anoms12 Instead of adding a toggle switch, holding the summary for 2-3 seconds will lock it. It'll stay open by default to avoid adding more strings. Could you open a feature request for it so I can keep track of what to include in the next release?

**Reply by 12th-devs** [reply]
Done
